.env

```diff
@@ -46,12 +46,12 @@ LOCAL_DB_TYPE=mysql
 ##
 # The database version to use.
 #
-# Defaults to 5.7 with the assumption that LOCAL_DB_TYPE is set to `mysql` above.
+# Defaults to 8.0 with the assumption that LOCAL_DB_TYPE is set to `mysql` above.
 #
 # When using `mysql`, see https://hub.docker.com/r/amd64/mysql for valid versions.
 # When using `mariadb`, see https://hub.docker.com/r/amd64/mariadb for valid versions.
 ##
-LOCAL_DB_VERSION=5.7
+LOCAL_DB_VERSION=8.0
```

Comment:
This seems even more timely now that the 5.7 image [seems to have been removed from the Docker registry altogether](https://core.trac.wordpress.org/ticket/59930#comment:16):

> I just checked https://hub.docker.com/r/amd64/mysql (which is what we're referring to from the [`.env` file](https://core.trac.wordpress.org/browser/trunk/.env?rev=56453#L51), and I noticed that MySQL 5.7 isn't even listed there anymore.
>
> I dug a bit deeper and found that it was [removed recently](https://github.com/docker-library/official-images/commit/3b2ef60353adbcee746512bc0f29f71967bf5bb9); the [commit](https://github.com/docker-library/mysql/commit/3e959c2) is titled "Remove 5.7 entirely (EOL)".